src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +55,26 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const TabItems = styled.div`
```

Comment:
You should manage the "selected" case, ie when a tab is `selected`, add a `border-bottom`

Reply:
Huumm... I'm using a border-bottom as hover, should i change ?

Reply:
Yes you should keep it when hover as well ;)